logger/empty_logger.go

```diff
@@ -0,0 +1,47 @@
+package logger
+
+type emptyLogger struct{}
```

Comment:
If only use in this package, keep initials lowercase is enough.

Better to name it as `default` and use `defalutlogger` in other places.

Reply:
?

Reply:
@1046102779